third_party/closure-responding-channel/closure-bundle.js

```diff
@@ -1,69 +1,65 @@
-a:{var ya="",za=function(){var a=v;if(va)return/rv:([^\);]+)(\)|;)/.exec(a);if(ua)return/Edge\/([\d\.]+)/.exec(a);if(ta)return/\b(?:MSIE|rv)[: ]([^\);]+)(\)|;)/.exec(a);if(wa)return/WebKit\/(\S+)/.exec(a);if(sa)return/(?:Version)[ \/]?(\S+)/.exec(a)}();za&&(ya=za?za[1]:"");if(ta){var x,Aa=m.document;x=Aa?Aa.documentMode:void 0;if(null!=x&&x>parseFloat(ya)){xa=String(x);break a}}xa=ya}var Ba=xa;function Ca(a){function b(f,g){var h=g+"  ";try{if(void 0===f)c.push("undefined");else if(null===f)c.push("NULL");else if("string"===typeof f)c.push('"'+f.replace(/\n/g,"\n"+g)+'"');else if("function"===typeof f)c.push(String(f).replace(/\n/g,"\n"+g));else if(p(f)){f[q]||d.push(f);var l=Object.prototype.hasOwnProperty.call(f,q)&&f[q]||(f[q]=++ba);if(e[l])c.push("*** reference loop detected (id="+l+") ***");else{e[l]=!0;c.push("{");for(var z in f)"function"!==typeof f[z]&&(c.push("\n"),c.push(h),c.push(z+
-" = "),b(f[z],h));c.push("\n"+g+"}");delete e[l]}}else c.push(f)}catch(Kb){c.push("*** "+Kb+" ***")}}var c=[],d=[],e={};b(a,"");for(a=0;a<d.length;a++)aa(d[a])};function y(a){a&&"function"==typeof a.R&&a.R()};var Da=function(){if(!m.addEventListener||!Object.defineProperty)return!1;var a=!1,b=Object.defineProperty({},"passive",{get:function(){a=!0}});try{m.addEventListener("test",n,b),m.removeEventListener("test",n,b)}catch(c){}return a}();function A(){this.h=this.h;this.l=this.l}A.prototype.h=!1;A.prototype.R=function(){this.h||(this.h=!0,this.o())};A.prototype.o=function(){if(this.l)for(;this.l.length;)this.l.shift()()};function B(a,b){this.type=a;this.g=this.target=b;this.defaultPrevented=!1}B.prototype.h=function(){this.defaultPrevented=!0};function C(a,b){B.call(this,a?a.type:"");this.relatedTarget=this.g=this.target=null;this.button=this.screenY=this.screenX=this.clientY=this.clientX=0;this.key="";this.metaKey=this.shiftKey=this.altKey=this.ctrlKey=!1;this.state=null;this.pointerId=0;this.pointerType="";this.i=null;if(a){var c=this.type=a.type,d=a.changedTouches&&a.changedTouches.length?a.changedTouches[0]:null;this.target=a.target||a.srcElement;this.g=b;if(b=a.relatedTarget){if(va){a:{try{ra(b.nodeName);var e=!0;break a}catch(f){}e=
+var l=this||self;function m(){}function n(a){var b=typeof a;return"object"==b&&null!=a||"function"==b}function aa(a){return Object.prototype.hasOwnProperty.call(a,p)&&a[p]||(a[p]=++ba)}function ca(a){null!==a&&"removeAttribute"in a&&a.removeAttribute(p);try{delete a[p]}catch(b){}}var p="closure_uid_"+(1E9*Math.random()>>>0),ba=0;function da(a,b,c){return a.call.apply(a.bind,arguments)}
```

Comment:
Reviewed, and this doesn't introduce anything suspicious.

Reply:
Awesome, thanks!